### DISCUSSION TITLE
Assimp and arcore on android

### DISCUSSION THREAD

**Discussion by armanTabib** [assimp/assimp#5574] [Q&A] [answered]
I have an application on android that uses arcore to do AR stuff. Now I want to expand my application and import other formats of 3d objects (other than .obj files). So I found assimp and I'm thinking of integrating it into my project. Is it feasible to load objects from assimp and import into arcore? Do you know any example projects?

**Comment by kimkulling** [comment]
Sure, you need to use the NDK to get assimp connected by your app.

Which kind of use cases do you want to address?

**Comment by tellypresence** [comment, marked as answer]
Have a look at official ARCore project [`hello_ar_c`](https://github.com/google-ar/arcore-android-sdk/tree/main/samples/hello_ar_c)

You'll need to replace the `.obj` model loading with your custom code
